CMakeLists.txt

```diff
@@ -11,11 +11,9 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
We should think critically before doing this and make sure it's necessary, I'd like to baseline on the lowest possible revision.